pkg/query-service/app/integrations/thirdPartyApi/translator.go

```diff
@@ -163,18 +170,22 @@ func BuildDomainList(thirdPartyApis *ThirdPartyApis) (*v3.QueryRangeParamsV3, er
 				Type:     v3.AttributeKeyTypeTag,
 			},
 		}, thirdPartyApis.GroupBy),
-		ReduceTo: v3.ReduceToOperatorAvg,
+		ReduceTo:  v3.ReduceToOperatorAvg,
+		ShiftBy:   0,
+		IsAnomaly: false,
 	}
 
-	builderQueries["error_rate"] = &v3.BuilderQuery{
-		QueryName:         "error_rate",
+	builderQueries["error"] = &v3.BuilderQuery{
+		QueryName:         "error",
 		DataSource:        v3.DataSourceTraces,
 		StepInterval:      defaultStepInterval,
-		AggregateOperator: v3.AggregateOperatorRate,
+		AggregateOperator: v3.AggregateOperatorCountDistinct,
 		AggregateAttribute: v3.AttributeKey{
-			Key: "",
+			Key:      "span_id",
+			DataType: v3.AttributeKeyDataTypeString,
+			IsColumn: true,
```

Comment:
Distinct is single thread. Unless it's a must, please prefer to use alternatives. In this case, CountDistinct(span_id) would be the same as Count, as each row corresponds to a span already.